CONTRIBUTORS.md

```diff
@@ -217,3 +217,4 @@
  - [olsh](https://github.com/olsh)
  - [lbenini](https://github.com/lbenini)
  - [gnuyent](https://github.com/gnuyent)
+ - [vgambier](https://github.com/vgambier)
```

Comment:
This section is contributors to the Emby code before Jellyfin was forked off, move this to above the "#Emby Contributors" line.

Reply:
Apologies. I force-pushed the change,